deploy/redis/README.md

```diff
@@ -0,0 +1,13 @@
+# Redis
+
+* The open source, in-memory data store used by millions of developers as a database, cache, streaming engine, and message broker.
```

Comment:
replace the line number 3 with below well-structured content
Redis is an open-source, in-memory data store widely used by developers as a database, cache, streaming engine, and message broker. In Mimoto, Redis is utilized as a cache store to enhance performance and speed up data retrieval.